### PR TITLE
Portfolio - mobile menu

### PR DESCRIPTION
**Pull request by Godedict**
Portfolio - mobile menu project was implemented using the following requirements:

- In mobile, implement the following features:

- When the user clicks (or taps) the hamburger button, the mobile menu appears.

- When the user clicks (or taps) the close (X) button, the mobile menu disappears.

- When the user clicks (or taps) any of the mobile menu options, the mobile menu disappears.

- When the user clicks (or taps) any of the mobile menu options, a correct part of the page is displayed.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -14,17 +14,24 @@
...
+          <a href="#">
+            <p>About</p>
+          </a>
         </li>
-        <li>
-          <a href="#"><p>Contact</p></a>
+        <li class="nav-item">
+          <a href="#">
+            <p>Contact</p>
+          </a>
```
Comment (levy002) [inline index.html]: - [ ] Kindly make sure when the user clicks on the navigation link the user is redirected to the corresponding part of the page. For example if the `Contact` nav link is clicked the page should be redirected to the Contact section, kindly fix that to all links.

**Review by levy002 — CHANGES_REQUESTED** [review]
## Hello @Godedict   , :wave:
### I'm Levy, your PR reviewer

Good job so far :+1: 
There are some issues that you still need to work on to go to the next project but you are almost there!



## Status: REQUIRED CHANGES ♻️
_Check the comments under the review._


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @levy002 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (Godedict) [reply index.html]: Thank you for the feedback. Changes implemented.

**Review by Qoosim — APPROVED** [review]
#  *Status: Approved* :tada: :heavy_check_mark: 

Hi @Godedict, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
-  Good README file. :heavy_check_mark: 
-  Correct GitHub Flow. :heavy_check_mark: 



## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**